spec/requests/api/v1/api_v1_partner_requests_spec.rb

```diff
@@ -84,7 +84,7 @@
         it "should only return items that are visible" do
           org = create(:organization, skip_items: true)
           item1 = create(:item, organization: org, visible_to_partners: true)
-          item2 = create(:item, organization: org, visible_to_partners: false)
+          item2 = create(:item, organization: org, visible_to_partners: false) # rubocop:disable Lint/UselessAssignment
```

Comment:
why not just `create(:item, organization: org, visible_to_partners: false)`

Reply:
That works. Updated.